src/lib/fast_check.py

```diff
@@ -76,7 +76,7 @@ def valid_dynatrace_scopes(token_metadata: dict):
 async def get_dynatrace_token_metadata(dt_session: ClientSession, context: LoggingContext, dynatrace_url: str, dynatrace_api_key: str, timeout: Optional[int] = 2) -> dict:
     try:
         response = await dt_session.post(
-            url=f"{dynatrace_url.rstrip('/')}/api/v1/tokens/lookup",
+            url=f"{dynatrace_url.rstrip('/')}/api/v2/apiTokens/lookup",
```

Comment:
Does `v2/apiTokens` support tokens in old format?

Reply:
It does. I've chosen but the output of v2 is more readable.

Reply:
Reverted to v1 as the name of the token is the only needed thing.